File: Algorithms/python/searching algorithms/linearSearch.py
def linear_Search(list1, n, key):  
  
    
    for i in range(0, n):  
        if (list1[i] == key):  
            return i  
    return -1  
  
  
list1 = list(map(int,input("Enter your list: ").split(' ')))  
key = int(input("enter key to search: "))
  
n = len(list1)  
res = linear_Search(list1, n, key)  
if(res == -1):  
    print("Element not found")  
else:  
    print("Element found at index: ", res) 

File: Algorithms/C/Searching Algorithms/Linear-search.c
#include<stdio.h>

int main()
{
    int array[100];
    int i, find, number;

    printf("\nEnter the number of elements in your array:- ");
    scanf("%d", &number);

    printf("\nEnter the array elements with SPACE as separator:- ");
    for (i = 0; i < number; ++i)
        scanf("%d", &array[i]);

    printf("\nEnter element to be searched:- ");
    scanf("%d", &find);

    for (i = 0; i < number; ++i)
        if (array[i] == find)
            break;

    if (i < number)
        printf("\nThe Element %d found at index %d", find, i);
    else
        printf("\nThe Element %d is not found", find);

    return 0;
}